### PR TITLE
Update Zigate-Plugin-FAQ.md

### PR DESCRIPTION
**Pull request by pipiche38**

### CODE REVIEW

File: en-eng/Zigate-Plugin-FAQ.md [hunk 1]
```diff
@@ -11,4 +11,12 @@ This is more an OS issue, but here are few hints
 1. Start the Plugin
 1. Trigger the pairing process from the device
 
+## I don't get the device updated in Domoticz when interacting via a remote control or manualy
+
+If I power Off an Ikea Bulb (with a physical switch), when switching it on I don't have a status updated in Domoticz. I have the same issue if using a remote controller .
```
Comment (karstenbakker) [inline en-eng/Zigate-Plugin-FAQ.md]: _For example :_ If you power off an Ikea bulb (with the physical switch) and then switch it on again the status of the bulb is not updated in Domoticz. and are having the same issue with a remote controller which also doesn't update it's current state to Domoticz.

If you encounter problems like the above example, it is most likely the automatic reporting is not configured correctly from the device end.  Please follow below steps to fix this issue:

File: en-eng/Zigate-Plugin-FAQ.md [hunk 2]
```diff
@@ -11,4 +11,12 @@ This is more an OS issue, but here are few hints
...
+
+Most likely the issue is that the automatic reporting is not correctly configured at the device end.
```
Comment (karstenbakker) [inline en-eng/Zigate-Plugin-FAQ.md]: See above example

File: en-eng/Zigate-Plugin-FAQ.md [hunk 3]
```diff
@@ -11,4 +11,12 @@ This is more an OS issue, but here are few hints
 1. Start the Plugin
 1. Trigger the pairing process from the device
 
+## I don't get the device updated in Domoticz when interacting via a remote control or manualy
+
+If I power Off an Ikea Bulb (with a physical switch), when switching it on I don't have a status updated in Domoticz. I have the same issue if using a remote controller .
+
+Most likely the issue is that the automatic reporting is not correctly configured at the device end. 
+
+1. Make sure that you have Allow reBinding Cluster in the PluginConf.txt ( that is the default ). check the parameters allowReBindingClusters it must be set to 1.
```
Comment (karstenbakker) [inline en-eng/Zigate-Plugin-FAQ.md]: 1. Open the PluginConf.txt file (more information about this here  .....)
2. Please check if the parameter "AllowRebindingClusters" is set to **1**.
_If it is not, set it to "1_"

File: en-eng/Zigate-Plugin-FAQ.md [hunk 4]
```diff
@@ -11,4 +11,12 @@ This is more an OS issue, but here are few hints
...
+1. Switch off the Ikea bulb with the physical switch, and then switch it on. It will re-do the rebinding and the configuration reporting.
```
Comment (karstenbakker) [inline en-eng/Zigate-Plugin-FAQ.md]: If you changed the parameter from 0 to 1, please make sure to restart the plugin so the settings will be enforced.

3. Switch off the Ikea bulb with the physical switch, wait for 10 seconds, then turn it on again. 
The rebinding process will start itself, the status problems should now be fixed.

Comment (pipiche38) [reply en-eng/Zigate-Plugin-FAQ.md]: fine

**Pull request merged**